### DISCUSSION TITLE
unable to clone with ssh key

### DISCUSSION THREAD

**Discussion by shermeenspotixx** [community/community#122000] [Code Security] [answered]
### Select Topic Area

Question

### Body

I recentry got access to github repository , I am trying to clone one respo in my machine and have crerated new ssh key pasted in git hub settings but when i run clone command it just says no access

**Comment by markolalovic** [comment, marked as answer]
Hi, you can try to [check your SSH access](https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors#check-your-ssh-access). 

Can you also provide the output from your terminal? It may be simpler to guess what's causing this.

**Reply by shermeenspotixx** [reply]
Yes it works for me!
